BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -360,11 +360,11 @@ extension AttributedLabel {
                     .paragraphStyle,
                     in: textStorage.entireRange,
                     options: []
-                ) { style, range, continuePointer in
+                ) { style, range, stop in
                     if let style = style as? NSParagraphStyle, range == textStorage.entireRange {
                         alignment = style.alignment
-                        continuePointer.pointee = false
                     }
+                    stop.pointee = true
```

Comment:
Sorry if I'm being dense, but why are we stopping outside of the conditional?

Reply:
If multiple paragraph styles are set over the label's entire range, we'll never enter the `if` clause, as none of them will be set over the entire range.

This means we can stop after hitting the first paragraph range. If we don't enter the `if` clause, we never will.